qa/standalone/scrub/osd-scrub-repair.sh

```diff
@@ -6199,8 +6142,6 @@ function TEST_request_scrub_priority() {
 
     # Verify that the requested scrub ran first
     grep "log_channel.*scrub ok" $dir/osd.${primary}.log | grep -v purged_snaps | head -1 | sed 's/.*[[]DBG[]]//' | grep -q $pg || return 1
-
-    return 0
```

Comment:
Note that this is a (maybe benign) change to the semantics

Reply:
> Note that this is a (maybe benign) change to the semantics

Yes, it's benign since the previous check involving grep must succeed for the test to return 0.